libraries/chain/include/eosio/chain/finality/qc.hpp

```diff
@@ -195,16 +195,24 @@ namespace eosio::chain {
 
    // finalizer authority of strong, weak, or missing votes
    struct qc_vote_metrics_t {
+      struct fin_auth_ele {
```

Comment:
What does `ele` stand for?

Reply:
`ele` is a shorthand for `element`. Use another name?

Reply:
Ah. Maybe just `fin_auth`.

Reply:
Done by https://github.com/AntelopeIO/spring/pull/453/commits/14226307c739967d3b5096a384b6de601ac6448e